### PR TITLE
chore: update language of local compilation doc

### PR DESCRIPTION
**Pull request by fhanrath**
I've changed some phrasing in the local compilation part of the docs to (hopefully) improve legibility.

Feel free to provide feedback or request further changes.

### CODE REVIEW

**Comment by fhanrath** [comment]
One thing I've noticed is, that [docs/po/zh_CN.po](https://github.com/HaoboGu/rmk/blob/main/docs/po/zh_CN.po) and [docs/src/user_guide/2-2_local_compilation.md](https://github.com/HaoboGu/rmk/blob/main/docs/src/user_guide/2-2_local_compilation.md) are slightly out of sync.

Is this something you intend to continue supporting? I tried to keep my changes in both files consistent without altering the differences.

File: docs/po/zh_CN.po [hunk 1]
```diff
@@ -476,14 +475,14 @@ msgid ""
 "is to create your own keymap definition and convert it to `vial.json` "
 "following **[vial's doc here](https://get.vial.today/docs/porting-to-via."
 "html)**, and place it at the root of the firmware project, replacing the "
-"default one. RMK would do all the rest things for you."
+"default one. RMK wwill do all the rest for you."
```
Comment (HaoboGu) [inline docs/po/zh_CN.po]: wwill -> will

**Comment by HaoboGu** [comment]
> One thing I've noticed is, that [docs/po/zh_CN.po](https://github.com/HaoboGu/rmk/blob/main/docs/po/zh_CN.po) and [docs/src/user_guide/2-2_local_compilation.md](https://github.com/HaoboGu/rmk/blob/main/docs/src/user_guide/2-2_local_compilation.md) are slightly out of sync.
> 
> Is this something you intend to continue supporting? I tried to keep my changes in both files consistent without altering the differences.

`.po` is somehow hard to update/maintain, I'm considering to remove it.

File: docs/po/zh_CN.po (hunk 1 above)
Comment (fhanrath) [reply docs/po/zh_CN.po]: fixed

**Comment by HaoboGu** [comment]
thanks a lot!

**Pull request merged**